macos/FluentUI/Link/Link.swift

```diff
@@ -144,7 +144,11 @@ open class Link: NSButton {
 
 	@objc private func linkClicked() {
 		if let url = url {
-			NSWorkspace.shared.open(url as URL)
+			if #available(macOS 10.15, *) {
```

Comment:
Oh have we not upgraded minimum supported OS of fluentui to 10.15 yet? We probably should do that.

Reply:
Filed a bug!